lighthouse-core/test/results/sample_v2.json

```diff
@@ -6414,12 +6381,6 @@
         "duration": 100,
         "entryType": "measure"
       },
-      {
-        "startTime": 0,
-        "name": "lh:computed:LanternLargestContentfulPaint",
-        "duration": 100,
-        "entryType": "measure"
-      },
```

Comment:
Were we making an extra call to this that we didn't need to?

Reply:
No, it was necessary. The preload LCP image audit makes an explicit *lantern* LCP request in order to get the LCP graph but our sample results here uses observed metrics for LCP itself which never requests the lantern computed artifact.